grpc/src/lib.rs

```diff
@@ -124,3 +154,16 @@ impl Octree for GrpcOctree {
         Ok(result)
     }
 }
+
+#[cfg(test)]
```

Comment:
remove? I think we could start writing integration tests, i.e. bringing up an octree server with a on-disk octree and then sending rpc's back and forth. Open an issue for that?

Reply:
#103

Reply:
please remove commented out code too.